lib/shared/util.ts

```diff
@@ -0,0 +1,13 @@
+const nodeUtil = require("util");
```

Comment:
Can we switch these to the import ... from ... syntax? (maybe as separate PR if they are too many)